osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointRenderer.cs

```diff
@@ -6,43 +6,32 @@
 using osu.Framework.Bindables;
 using osu.Framework.Extensions;
 using osu.Framework.Graphics;
-using osu.Framework.Graphics.Containers;
-using osu.Framework.Graphics.Performance;
 using osu.Framework.Graphics.Pooling;
 using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Pooling;
 
 namespace osu.Game.Rulesets.Osu.Objects.Drawables.Connections
 {
     /// <summary>
     /// Visualises connections between <see cref="DrawableOsuHitObject"/>s.
     /// </summary>
-    public class FollowPointRenderer : CompositeDrawable
+    public class FollowPointRenderer : PooledDrawableWithLifetimeContainer<FollowPointLifetimeEntry, FollowPointConnection>
     {
-        public override bool RemoveCompletedTransforms => false;
-
-        public IReadOnlyList<FollowPointLifetimeEntry> Entries => lifetimeEntries;
+        public new IReadOnlyList<FollowPointLifetimeEntry> Entries => lifetimeEntries;
```

Comment:
Bit awkward how instances of this class will store two lists of lifetime entries (one from the base, and one local here). The local seems to have some ordering logic that probably can't go away, so I'm not sure much can be done about it without restructuring the base some more.

Reply:
I think this is fine for now. Many ways to restructure, but it was like this previously and that can be done in a future PR if desired.